x-pack/plugins/index_management/public/application/components/template_form/steps/step_review.tsx

```diff
@@ -268,6 +269,19 @@ export const StepReview: React.FunctionComponent<Props> = React.memo(
                 {getDescriptionText(serializedSettings)}
               </EuiDescriptionListDescription>
 
+              {/* Index mode */}
+              <EuiDescriptionListTitle data-test-subj="indexModeTitle">
+                <FormattedMessage
+                  id="xpack.idxMgmt.templateForm.stepReview.summaryTab.indexModeLabel"
+                  defaultMessage="Index mode"
+                />
+              </EuiDescriptionListTitle>
+              <EuiDescriptionListDescription data-test-subj="indexModeValue">
+                {getIndexModeLabel(
+                  serializedSettings?.['index.mode'] ?? serializedSettings?.index?.mode
```

Comment:
Is it the case that the index mode can be in two different object structures? If so, why?

Reply:
Yes, Es accepts both formats and so the Settings editor in the index template creation flow allows the user to use both formats as well.